internal/pkg/aws/ssm/ssm.go

```diff
@@ -0,0 +1,73 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package ssm provides a client to make API requests to Amazon Systems Manager.
+package ssm
+
+import (
+	"fmt"
+	"sort"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+type api interface {
```

Comment:
It looks like we have been using SSM through [store](https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/config/store.go#L49) -- is the plan was to refactor places where we are using that SSM client to use the new one? I remember @kohidave had reasons for wanting to abstract SSM back in the early copilot days so just wanted to make sure.

Reply:
My understanding was that the `config/store` package was used to store app, env or workload configurations (and we "happened" to have done it via ssm parameter store); while secret here is considered a ssm resource that gets created upon request - instead of a Copilot configuration. So I think we won't need to refactor `store` to use ssm client here. They can both exist and serve different purposes.

Reply:
But we should wait on other's input on this! Maybe there was other reasons to it 👂🏼

Reply:
I think it is probably fine to keep `config` as is since every pkg under `aws` should be agnostic about copilot 🤔